typescript/src/v0/Mcap0UnindexedWriter.ts

```diff
@@ -0,0 +1,60 @@
+import { IWritable } from "./IWritable";
+import { Mcap0RecordWriter } from "./Mcap0RecordWriter";
+import { ChannelInfo, Message, Header, Attachment } from "./types";
+
+export class Mcap0UnindexedWriter {
```

Comment:
I imagine indexed and unindexed writing could potentially be in the same class, with just a boolean constructor param to indicate whether we should be holding index info in memory and writing it at the end?